tracer/contrib/config.go

```diff
@@ -0,0 +1,40 @@
+package contrib
+
+import "fmt"
+
+var MYSQL_CONFIG = MySQLConfig{
```

Comment:
It's just for testing purposes, because when dd-trace-go will grow bigger, we'll want to import the config for mysql and postgres server from env variables (like in dd-trace-py: [.env](https://github.com/DataDog/dd-trace-py/blob/master/.env), [config.py](https://github.com/DataDog/dd-trace-py/blob/020e9d7a77cb685178509f3e4759bdfd2b26eaee/tests/contrib/config.py)).